game/scripts/npc/items/item_radiance_3.txt

```diff
@@ -1,5 +1,5 @@
-"DOTAItems"
-{
+"DOTAItems"
+{

```

Comment:
yeah fuck those line endings. just read what i wrote below in order to remove them using Sublime Text 3.